packages/fxa-auth-server/lib/senders/emails/partials/appBadges/en.ftl

```diff
@@ -4,5 +4,18 @@ body-android-badge = <img data-l10n-name="google-play-badge" alt="Download { $pr
 # Variables:
 #  $productName (String) - The name of the product to be downloaded, e.g. Mozilla VPN, or Firefox
 body-ios-badge = <img data-l10n-name="apple-app-badge" alt="Download { $productName } on the { -app-store }">
-another-desktop-device = Or, install on <a data-l10n-name="anotherDeviceLink">another desktop device</a>.
-another-device = Or, install on <a data-l10n-name="anotherDeviceLink">another device</a>.
+# Variables:
+#  $productName (String) - The name of the product to be downloaded, e.g. Mozilla VPN, or Firefox
+another-desktop-device-2 = Or, install { $productName } on <a data-l10n-name="anotherDeviceLink">another desktop device</a>.
+# Variables:
+#  $productName (String) - The name of the product to be downloaded, e.g. Mozilla VPN, or Firefox
+another-device-2 = Or, install { $productName } on <a data-l10n-name="anotherDeviceLink">another device</a>.
+# Variables:
+#  $productName (String) - The name of the product to be downloaded, e.g. Mozilla VPN, or Firefox
+android-download-plaintext = Get { $productName } on Google Play:
+# Variables:
+#  $productName (String) - The name of the product to be downloaded, e.g. Mozilla VPN, or Firefox
+ios-download-plaintext = Get { $productName } on the App Store:
+# Variables:
+#  $productName (String) - The name of the product to be downloaded, e.g. Mozilla VPN, or Firefox
+another-device-plaintext = Get { $productName } on another device:
```

Comment:
Perhaps test coverage on these new strings should be added? Note, there is currently test coverage on `body-ios-badge` and  `body-android-badge` l10n strings.